.github/ISSUE_TEMPLATE/new-repo-request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Repository creation in cncf-tags org
+description: Create a repository in the cncf-tags GitHub Org
+title: "[REPO CREATION] Create <github-repo> in cncf-tags"
+body:
+- id: repo_name
+  type: input
+  attributes:
+    label: Requested name for new repository
+    placeholder: e.g. container-device-interface
+  validations:
+    required: true
+- id: description
+  type: textarea
+  attributes:
+    label: What should the repo description be?
+  validations:
+    required: false
+- id: tag
+  type: input
+  attributes:
```

Comment:
If you want, you can also use a dropdown to have it more consistently reported.

For example: https://github.com/kedacore/keda/blob/main/.github/ISSUE_TEMPLATE/3_bug_report.yml?plain=1#L55-L84

However, this might be intentional

Reply:
Also added https://github.com/cncf/toc/pull/1101

Reply:
> However, this might be intentional

I thought about this, then figured that adding a new dropdown value when a new TAG or WG is formed might introduce a maintaince burden.

We can see how this textbox input method goes and if it isn't helpful, we can introduce a dropdown menu here.